Makefile

```diff
@@ -320,7 +320,7 @@ install-deps-ALPINE-for-pysemgrep:
 # - pkg-config?
 # - coreutils?
 # - gettext?
-BREW_DEPS=pcre gmp pkg-config coreutils gettext
+BREW_DEPS=pcre gmp pkg-config coreutils gettext libev
```

Comment:
there is an UBUNTU_DEPS also now in this Makefile you could also update.

Reply:
will do